app/views/govuk_publishing_components/components/_intervention.html.erb

```diff
@@ -0,0 +1,24 @@
+<%
+  local_assigns[:margin_bottom] ||= 8
+  local_assigns[:margin_bottom] = 8 if local_assigns[:margin_bottom] > 9
+
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+
+  css_classes = %w[gem-c-intervention]
+  css_classes << (shared_helper.get_margin_bottom)
+
+  aria_attributes = {label: 'Intervention'}
```

Comment:
Is 'intervention' a word that users will understand? I'm asking because I'm not sure I understand it in this context.

Reply:
probably not, that's why I'm removing aria-label (see above)